src/test/java/com/salesforce/dynamodbv2/mt/mappers/MTAmazonDynamoDBTestRunner.java

```diff
@@ -30,8 +30,10 @@
 import com.amazonaws.services.dynamodbv2.model.QueryRequest;
 import com.amazonaws.services.dynamodbv2.model.ScalarAttributeType;
 import com.amazonaws.services.dynamodbv2.model.ScanRequest;
+import com.amazonaws.services.dynamodbv2.model.ScanResult;
 import com.amazonaws.services.dynamodbv2.model.StreamRecord;
 import com.amazonaws.services.dynamodbv2.model.UpdateItemRequest;
+import com.amazonaws.services.dynamodbv2.xspec.N;
```

Comment:
my local copy of this branch shows this import not being used

Reply:
Thanks, will fix